CHANGELOG.md

```diff
@@ -1,3 +1,246 @@
+## v0.5.0 - 2020-10-23
+
+We're happy to announce the release of Lokomotive v0.5.0 (Eurostar).
+
+This release packs new features, bug fixes, code optimizations, platform updates, security hardening, and much more.
+
+### Changes in v0.5.0
+
+#### Kubernetes updates
+
+- Kubernetes version has been updated to
+  [`v1.19.3`](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md#v1193)
+  ([#1030](https://github.com/kinvolk/lokomotive/pull/1030)).
+
+#### Platform updates
+
+##### AKS
+
+- Kubernetes version has been updated to `1.18.8`
+  ([#1071](https://github.com/kinvolk/lokomotive/pull/1071)).
+
+##### Baremetal
+
+- Expose CNI MTU on the baremetal platform
+  ([#977](https://github.com/kinvolk/lokomotive/pull/977)).
+
+#### New components
+
+- Component `web-ui`
+  ([#981](https://github.com/kinvolk/lokomotive/pull/981)),
+  ([#1100](https://github.com/kinvolk/lokomotive/pull/1100))
+  from [headlamp](https://github.com/kinvolk/headlamp).
+- Component `inspektor-gadget`
+  ([#1076](https://github.com/kinvolk/lokomotive/pull/1076))
+  from [inspektor-gadget](https://github.com/kinvolk/inspektor-gadget/).
+
+#### Component updates
+
+- Update Velero component for Packet [OpenEBS and restic plugin support]
+  ([#881](https://github.com/kinvolk/lokomotive/pull/881)).
+- istio-operator: Update to 1.7.3
+  ([#1086](https://github.com/kinvolk/lokomotive/pull/1086)).
+- prometheus-operator: Update grafana, kube-state-metrics and node_exporter
+  ([#963](https://github.com/kinvolk/lokomotive/pull/963)).
+
+#### Terraform updates
+
+- Upgrade to Terraform 0.13
+  ([#824](https://github.com/kinvolk/lokomotive/pull/824)).
+
+#### Features
+
+- Encryption support to in-cluster pod traffic
+  ([#911](https://github.com/kinvolk/lokomotive/pull/911)).
+- AWS, Packet, Baremetal: use Docker instead of rkt for host containers
+  ([#946](https://github.com/kinvolk/lokomotive/pull/946)).
+- Change labels and taints format from string to structured
+  ([#1042](https://github.com/kinvolk/lokomotive/pull/1042)).
+- prometheus-operator: Add external_url
```

Comment:
```suggestion
- Prometheus-operator: Add external_url
```

Reply:
`prometheus-operator` is the name of the component. I think it's OK.